content/code_samples/escrow/websocket/tx-request-escrowfinish-time.json

```diff
@@ -0,0 +1,5 @@
+{
+  "id": 21,
```

Comment:
When the `id` here is `21`, the response will also have an `id` of `21`. Currently, it says `2`

https://github.com/ripple/ripple-dev-portal/pull/265/files#diff-63ca9a6ad7dbc7362b541a73a5c71a2fR2

Reply:
Good catch. Fixed in c479843